app/src/main/java/com/realityexpander/tasky/MainActivity.kt

```diff
@@ -40,6 +48,53 @@ class MainActivity : ComponentActivity() {
     }
 }
 
+@Composable
+@Destination
+@RootNavGraph(start = true)
+fun SplashScreen(
```

Comment:
I'd avoid using your own Splash screens if it's not a very complex one. Android already offers a built in splash screen API which will always show a splash screen for devices running on 11+. If you implement your own one, it might show 2 splash screens on some devices.
Android's API also allows you to keep the splash screen visible until a certain condition is met (like the user being authenticated). In this video I explain it:
https://www.youtube.com/watch?v=Loo4i5IrZ4Y

Reply:
OK, i see this method is much easier than what I was doing. But Im still curious how the two screen would show on some devices, as im just using the `DestinationsNavhost` and navigating from screen to screen.